### PR TITLE
HttpBuildTimeConfig - add configurable debug path

### PR DESCRIPTION
**Pull request by mkouba**
- defaults to "/quarkus"

### CODE REVIEW

**Comment by geoand** [comment]
Test failures seem related:

```
2020-05-25T11:48:37.6183285Z [ERROR]   ArcEndpointTest.testBeans:58 expected: <io.quarkus.runtime.StartupEvent> but was: <io.vertx.ext.web.Router>
2020-05-25T11:48:37.6183630Z [INFO] 
```

**Comment by mkouba** [comment]
@geoand very related indeed. The test is just wrong. Thanks for spotting this...

**Comment by mkouba** [comment]
@maxandersen ping ;-)

File: extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/devmode/ArcEndpointTest.java [hunk 1]
```diff
@@ -46,21 +50,28 @@ public void testBeans() {
...
-        assertEquals(StartupEvent.class.getName(), fooObserver.getString("observedType"));
-        assertEquals(2500, fooObserver.getInteger("priority"));
     }
 
     @Named
     @ApplicationScoped
     public static class Foo {
 
+        @Inject // do not remove the bean
+        HttpBuildTimeConfig httpConfig;
```
Comment (geoand) [inline extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/devmode/ArcEndpointTest.java:67]: I'm a little unclear about this, can you please explain it?

Comment (mkouba) [reply extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/devmode/ArcEndpointTest.java:67]: Ah, that's a leftover from the previous version of the test where I tried to inject the config root using `Arc.container().instance()` (and it's removable). Will remove the comment...

Comment (geoand) [reply extensions/vertx-http/deployment/src/test/java/io/quarkus/vertx/http/devmode/ArcEndpointTest.java]: Cool, because I was beginning to worry :)

**Review by geoand — APPROVED** [review]

**Review by gsmet — COMMENTED** [review]
So there's one thing I'm not totally sure of, the naming.

I think at some point we will have a complete dev console at this path (at least that's what I would like). So calling it `debug` is a bit limiting and a bit unclear to me (not talking about the endpoint, `quarkus` is fine but the config properties).

Is `console` clear enough in English? Or `devConsole`?

**Comment by mkouba** [comment]
Yes, I'm also not sure about the name but I expect users to rarely modify the value... I was reluctant to use anything with `dev` because that would suggest it's only available in the dev mode which might not be the case in the future.

**Comment by stuartwdouglas** [comment]
I prefer console.

**Comment by mkouba** [comment]
Ok, be it `quarkus.http.consolePath` then. Any objections?

**Pull request merged**